fastlane/Fastfile

```diff
@@ -1,17 +1,5 @@
-lane :test do
```

Comment:
`m`: Shouldn't we keep the old tests until we remove the APIs? They don't do any harm and validate that the old way is still working.

Reply:
I think this particular test was never even called, thats why I removed it. Will check again.

Reply:
Yeah we never used `test` and `test_big` in CI. Also `./assets/Big.app.dSYM.zip` does not exist.